src/Record/Traits.php

```diff
@@ -66,6 +66,9 @@
  * @property-read string|null $organization The name of the organization associated
  * with the IP address. This attribute is only available from the City and
  * Insights web services and the GeoIP2 Enterprise database.
+ * @property-read int|null $userCount <p>The estimated number of users sharing
+ * the IP/network during the past 24 hours. For IPv4, the count is for the
+ * individual IP. For IPv6, the count is for the /64 network.</p>
```

Comment:
Get rid of the `<p>`; it was needed below because of the list.

Also mention that this is only for the Insights web service.